sector-builder-ffi/Cargo.toml

```diff
@@ -16,9 +16,12 @@ crate-type = ["rlib", "cdylib", "staticlib"]
 [dependencies]
 drop_struct_macro_derive = "0.4.0"
 ffi-toolkit = "0.4.0"
-filecoin-proofs = { git = "https://github.com/filecoin-project/rust-fil-proofs.git", branch = "master" }
-storage-proofs = { git = "https://github.com/filecoin-project/rust-fil-proofs.git", branch = "master" }
-filecoin-proofs-ffi = { git = "https://github.com/filecoin-project/rust-fil-proofs-ffi.git", branch = "master" }
+# FIXME: This should be master branch or a version
+filecoin-proofs = { git = "https://github.com/filecoin-project/rust-fil-proofs.git", rev = "b7e29a5e739df3235823e16cd8e4ee1a1089d14a" }
```

Comment:
BLOCKING: As you've called out in the FIXME, these need to be updated to point to `master` once those upstream repos have the appropriate merkle tree codes.